static/css/style.css

```diff
@@ -0,0 +1,6 @@
+#username-form{
+    display: none; 
+}
+.red{
```

Comment:
@lamngue let's have only one css file for the whole project. You can move these lines to the file `pachimari.css` and include it in. This is the common css file that is in use by the rest of the project